.github/workflows/prerelease_linux.yml

```diff
@@ -17,10 +17,10 @@ env:
   TAG:  ${{ github.event.release.tag_name }}
   DOCKER_HUB_ID: ${{ secrets.OHAI_DOCKER_HUB_ID }}
   DOCKER_HUB_PASSWORD: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
-  #TODO change ci/pipeline to real branch name
-  SCHEMA_BRANCH: "ci/pipeline"
-  AWS_S3_BUCKET_NAME: "nr-downloads-ohai-staging"
-  AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock-staging"
+  #TODO change ci/pipeline-arm-from-master to real branch name
+  SCHEMA_BRANCH: "ci/pipeline-arm-from-master"
```

Comment:
we could replace *branches* with *commit hashes*, as this https://raw.githubusercontent.com/newrelic/infrastructure-agent/fa8438b46cadb13586107010a0dae14ba9302a26/.github/workflows/prerelease_linux.yml
will live forever, but a branch won't

Reply:
this already happened https://github.com/newrelic/infrastructure-agent/runs/2395390838?check_suite_focus=true

Reply:
it should be master or feature/dev branch if you want to try some changes in it.